client/jsconfig.json

```diff
@@ -3,7 +3,9 @@
     "checkJs": true,
     "target": "es2020",
     "module": "es2020",
-    "strict": true
+    "strictPropertyInitialization": true,
+    "strictNullChecks": true,
+    "strictBindCallApply": true
```

Comment:
Same here, these types of changes should go in a separate PR in the future, since they're not relevant to the purpose of this PR.